cmd/common.go

```diff
@@ -62,7 +62,17 @@ func PrepareEnvironmentAndRunCommand(userConfig config.UserConfigModel, cmdName
 
 	origGOPATH := os.Getenv("GOPATH")
 	if origGOPATH == "" {
-		return 0, fmt.Errorf("You don't have a GOPATH environment - please set it; GOPATH/bin will be symlinked")
+		// since Go 1.8 GOPATH is no longer required, it defaults to $HOME/go if not set:
+		// https://golang.org/doc/go1.8#gopath
+		p, err := pathutil.AbsPath("$HOME/go")
+		if err != nil {
+			return 0, errors.Wrap(err, "No GOPATH environment variable specified, and failed to get Abs path of default $HOME/go dir")
+		}
+		origGOPATH = p
```

Comment:
Why not?
```
origGOPATH := os.Getenv("HOME")+"/go"
```

Reply:
True - I guess it's more of a habit, as I believe AbsPath has all kinds of normalization which helps with user defined paths (I wrote it a long while ago :P ), but indeed here `$HOME` can't really be a custom path.